gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.58.11
+version=29.58.12
```

Comment:
Done

Reply:
Please see my other comment emitting SD event. It will need some changes in the callbacks of various markup/down in ZookeeperAnnouncer.

Reply:
>It will need some changes in the callbacks of various markup/down in ZookeeperAnnouncer.
How about directly change `emitSDStatusActiveUpdateIntentAndWriteEvents` ? Updated